### DISCUSSION TITLE
Images and Github Metrics not displaying

### DISCUSSION THREAD

**Discussion by MonitSharma** [community/community#42693] [Other Feature Feedback, Questions, & Ideas] [answered]
### Github Profile README

Not able to view images and gifs in Github Readme!

### Body

I have made my Github Readme file with some gifs and images that are stored in the repository `MonitSharma`, earlier they used to display properly on my Github Profile, but from some time back ,the images and the gifs are not loading properly. The files are not so big in size and the path length is short as well, I am not sure what's causing the problem.

I also use Github Actions to display my Github metrics on the the profile page, it generates an `.svg` , but that is also not being able to load properly. I am not sure what's causing the problem. 

Kindly Help me regarding this.

**Comment by jfullstackdev** [comment]
can you send the link ? so I can check whether it will be loading the first time I visit it 

you can view that in Incognito without log in, maybe cached data is what is happening in your regular tab

**Reply by MonitSharma** [reply]
https://github.com/MonitSharma

I have tried seeing it through incognito. It still doesn't load.

**Reply by jfullstackdev** [reply]
it loaded fine on me, so it's your end, try pinning it down, maybe ISP, firewall, antivirus, low memory etc 

![image](https://user-images.githubusercontent.com/47092464/210139077-9f8248f6-3505-44cd-bef8-bc2d16a09f38.png)

**Reply by MonitSharma** [reply]
Thank you for this, the issue is indeed from my side. But I am not able to rectify it. I tried different browsers, tried to look from my mobile as well, changed the Internet Service Provider, but I'm glad it's visible to everyone else.

**Comment by Abubakrce19** [comment]
There could be a few potential issues that could cause images or GIFs to not display properly on your GitHub profile. Here are a few things you can try to troubleshoot the issue:

Make sure that the file names of the images and GIFs are correct and that they are being referenced correctly in the Readme file.

Check to see if the images and GIFs are being hosted on a reliable server. If the server is down or experiencing issues, the images and GIFs may not display properly.

If you are using GitHub Actions to generate an .svg file, make sure that the action is being triggered correctly and that the .svg file is being generated and committed to the repository correctly.

If the problem persists, try clearing your cache and cookies and trying to load the images and GIFs again.

If none of these suggestions help, it could be helpful to provide more information about the specific error message or behavior you are experiencing. This will allow me to better understand the problem and provide more targeted recommendations.

Thanks

**Reply by MonitSharma** [reply]
>> Make sure that the file names of the images and GIFs are correct and that they are being referenced correctly in the Readme file.

Yes, I have checked they are correct.

>> Check to see if the images and GIFs are being hosted on a reliable server. If the server is down or experiencing issues, the images and GIFs may not display properly.

The images and gifs are all downloaded and are saved in the same github repository. So they are on Github only.

>> If you are using GitHub Actions to generate an .svg file, make sure that the action is being triggered correctly and that the .svg file is being generated and committed to the repository correctly.

Cross checked this and everything is correct here as well.

>> If the problem persists, try clearing your cache and cookies and trying to load the images and GIFs again.

Did that too and also tried to see my profile from an incognito, mobile device and other devices as well.

>> If none of these suggestions help, it could be helpful to provide more information about the specific error message or behavior you are experiencing. This will allow me to better understand the problem and provide more targeted recommendations.

There is no error message, I have downloaded and saved the gifs and the images in the repository [here](https://github.com/MonitSharma/MonitSharma), I usually displayed them all in the the Github Readme, by simply using `![](img source)`, as I have done everytime. Check [here](https://github.com/MonitSharma/MonitSharma/blob/main/README.md)

Earlier it used to load everything and all the gifs were properly rendering and were able to show up in the readme file, or in the My profile section of my account.

But from quite some time the images are not being loaded, it doesn't show anything there, just some blank space, I deleted the images and reuploaded them , but to no avail.

I don't know why it is happening?

**Reply by Abubakrce19** [reply]
> > > Make sure that the file names of the images and GIFs are correct and that they are being referenced correctly in the Readme file.
> 
> Yes, I have checked they are correct.
> 
> > > Check to see if the images and GIFs are being hosted on a reliable server. If the server is down or experiencing issues, the images and GIFs may not display properly.
> 
> The images and gifs are all downloaded and are saved in the same github repository. So they are on Github only.
> 
> > > If you are using GitHub Actions to generate an .svg file, make sure that the action is being triggered correctly and that the .svg file is being generated and committed to the repository correctly.
> 
> Cross checked this and everything is correct here as well.
> 
> > > If the problem persists, try clearing your cache and cookies and trying to load the images and GIFs again.
> 
> Did that too and also tried to see my profile from an incognito, mobile device and other devices as well.
> 
> > > If none of these suggestions help, it could be helpful to provide more information about the specific error message or behavior you are experiencing. This will allow me to better understand the problem and provide more targeted recommendations.
> 
> There is no error message, I have downloaded and saved the gifs and the images in the repository [here](https://github.com/MonitSharma/MonitSharma), I usually displayed them all in the the Github Readme, by simply using `![](img source)`, as I have done everytime. Check [here](https://github.com/MonitSharma/MonitSharma/blob/main/README.md)
> 
> Earlier it used to load everything and all the gifs were properly rendering and were able to show up in the readme file, or in the My profile section of my account.
> 
> But from quite some time the images are not being loaded, it doesn't show anything there, just some blank space, I deleted the images and reuploaded them , but to no avail.
> 
> I don't know why it is happening?

It sounds like you have done a thorough check of your file paths, file formats, file sizes, and markdown syntax, and that the issue is not related to any of these factors.

If the images and gifs used to load properly on your Github profile but have stopped working recently, it could be that there is a problem with the hosting of the images or gifs on Github.

One thing you could try is to clear your browser cache and try loading the images and gifs again. This will ensure that you are seeing the most up-to-date version of the files.

If that doesn't help, you might want to try uploading the images and gifs to a different hosting service (such as Imgur or Cloudinary) and linking to them from your Github profile. This will help you determine whether the issue is with Github's hosting service or something else.

If the issue persists even when using a different hosting service, it could be that there is a problem with your internet connection or with your device. In this case, you might want to try accessing your Github profile from a different device or network to see if the issue persists.

I hope this helps! If you continue to have problems, please let me know and I will do my best to assist you further.

**Reply by MonitSharma** [reply]
>> One thing you could try is to clear your browser cache and try loading the images and gifs again. This will ensure that you are seeing the most up-to-date version of the files.

I have cleared the cache of my browser. Here's the link to [My Profile](https://github.com/MonitSharma), tell me are you able to view the pictures and gifs.


>> If that doesn't help, you might want to try uploading the images and gifs to a different hosting service (such as Imgur or Cloudinary) and linking to them from your Github profile. This will help you determine whether the issue is with Github's hosting service or something else.

I tried this as well, searched for a random image on imgur and tried to display it on the Readme page, still the same issue.

>>If the issue persists even when using a different hosting service, it could be that there is a problem with your internet connection or with your device. In this case, you might want to try accessing your Github profile from a different device or network to see if the issue persists.

I have tried accessing my profile from different devices, accounts, and even ISPs, but to no avail.

As I said, everything was normal around a month back, but then suddenly it changed.

**Comment by AbubakrChan** [comment]
I thin you should mark him
 as an answer .. just suggesting

**Comment by AbubakrChan** [comment]
@MonitSharma

**Comment by THETURTLENECK** [comment]
I'm having the same problem. I can not see any images in my github. Has anyone solved it, please help.

![Screenshot_20230127_053445](https://user-images.githubusercontent.com/106016011/215082553-12b79e5e-ceb9-461e-92a8-bdb26fc09d41.png)
![Screenshot_20230127_053520](https://user-images.githubusercontent.com/106016011/215082578-6f140f99-db16-4550-80b1-de991b83d649.png)

**Reply by MonitSharma** [reply]
Hi @THETURTLENECK , can you answer the below questions:

1) Is your Internet Service provider Jio?

2) Go to the [debug site](https://github-debug.com/) and follow the guidelines there and paste the output here

Thanks

**Reply by THETURTLENECK** [reply]
#### 1. Yes
#### 2. Here are the results of the debug test.
1. Local commands :
```
PS C:\Users\theon> git clone https://github.com/github/debug-repo debug-repo-http
Cloning into 'debug-repo-http'...
remote: Enumerating objects: 1653, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
Receiving objects: 100% (1653/1653), 329.28 KiB | 2.55 MiB/s, done.

Resolving deltas: 100% (1063/1063), done.
PS C:\Users\theon> git clone git@github.com:github/debug-repo debug-repo-ssh
Cloning into 'debug-repo-ssh'...
The authenticity of host 'github.com (20.207.73.82)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
PS C:\Users\theon> ping -n 10 github.com

Pinging github.com [20.207.73.82] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 20.207.73.82:
    Packets: Sent = 10, Received = 0, Lost = 10 (100% loss),
PS C:\Users\theon> tracert github.com

Tracing route to github.com [20.207.73.82]
over a maximum of 30 hops:

  1     1 ms     1 ms     1 ms  reliance.reliance [192.168.29.1]
  2     7 ms     3 ms     3 ms  10.13.216.1
  3     9 ms     4 ms     9 ms  172.16.18.1
  4     3 ms     4 ms     5 ms  192.168.128.136
  5     9 ms     4 ms     9 ms  172.27.248.53
  6     6 ms     4 ms     4 ms  172.27.248.35
  7     3 ms     4 ms     5 ms  192.168.44.28
  8     *        *        *     Request timed out.
  9     *        *        *     Request timed out.
 10     *        *        *     Request timed out.
 11     *        *        *     Request timed out.
 12     *        *        *     Request timed out.
 13    10 ms     9 ms    10 ms  reliancejio.del01-96cbe-1b.ntwk.msn.net [104.44.196.187]
 14    36 ms    33 ms    33 ms  ae20-0.del01-96cbe-1a.ntwk.msn.net [104.44.42.87]
 15    33 ms    31 ms    37 ms  ae32-0.ear01.bom02.ntwk.msn.net [104.44.41.136]
 16    36 ms    33 ms    32 ms  be-23-0.ibr03.bom02.ntwk.msn.net [104.44.34.35]
 17    43 ms    39 ms    38 ms  be-5-0.ibr01.pnq21.ntwk.msn.net [104.44.30.142]
 18    40 ms    39 ms    40 ms  ae106-0.rwa04.pnq21.ntwk.msn.net [104.44.20.68]
 19     *        *        *     Request timed out.
 20     *        *        *     Request timed out.
 21     *        *        *     Request timed out.
 22     *        *        *     Request timed out.
 23     *        *        *     Request timed out.
 24     *        *        *     Request timed out.
 25     *        *        *     Request timed out.
 26     *        *        *     Request timed out.
 27     *        *        *     Request timed out.
 28     *        *        *     Request timed out.
 29     *        *        *     Request timed out.
 30     *        *        *     Request timed out.

Trace complete. 
```

2. Connection data :
```                
2507998 bytes downloaded from github.com at 14.68 Mbps
2507998 bytes downloaded from cloud.githubusercontent.com at 6.51 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 68.83 Mbps
2507998 bytes downloaded from github.githubassets.com at 13.66 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 2.44 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 1.40 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 44.71 Mbps
2507998 bytes downloaded from fra.github-debug.com at 12.27 Mbps
2507998 bytes downloaded from iad.github-debug.com at 2.12 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 11.25 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 3.17 Mbps
2507998 bytes downloaded from sea.github-debug.com at 1.69 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 21.79 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 1.81 Mbps

                
49.36.190.200

                
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36

                
4906006a259b7e55b604ee5b15ad4ac4

                
DCCE:269C:05B6:53CC8:63D4AB42

                
                

                  
traceroute to 49.36.190.200 (49.36.190.200), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 49.36.190.200 (49.36.190.200) 56(84) bytes of data.

--- 49.36.190.200 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2035ms


                  
Start: 2023-01-27T20:57:53-0800
HOST: 311ba7fdeb3cc8e5b069f236b19dd063               Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
Host 200.190.36.49.in-addr.arpa. not found: 3(NXDOMAIN)
```

**Reply by THETURTLENECK** [reply]
Is there anything I can do to solve this?

**Reply by MonitSharma** [reply]
Jio's ISP is facing an issue with loading the images file, as you can see in the output, you're facing a 100% packet loss. My case was the same

Either you can contact your ISP about this, or change your ISP,
I shifted to Airtel and everything loads properly, because I have no idea how to contact JIO and address this issue with them.

**Reply by THETURTLENECK** [reply]
Ok Thanks 👍🏼

**Comment by MonitSharma** [comment, marked as answer]
For everyone, the problem has been solved. The problem was from my side and my Internet Service provider. 
While browsing through the [debug site](https://github-debug.com/) and following all the steps.

I did the local and connection test. I am writing/pasting the results here.

```
Monit Sharma@Monit-Sharma MINGW64 ~ (master)
$ git clone https://github.com/github/debug-repo debug-repo-http
tracert github.comfatal: destination path 'debug-repo-http' already exists and is not an empty directory.

Monit Sharma@Monit-Sharma MINGW64 ~ (master)
$ git clone git@github.com:github/debug-repo debug-repo-ssh
Cloning into 'debug-repo-ssh'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Monit Sharma@Monit-Sharma MINGW64 ~ (master)
$ ping -n 10 github.com

Pinging github.com [20.207.73.82] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 20.207.73.82:
    Packets: Sent = 10, Received = 0, Lost = 10 (100% loss),

Monit Sharma@Monit-Sharma MINGW64 ~ (master)
$ tracert github.com

Tracing route to github.com [20.207.73.82]
over a maximum of 30 hops:

  1     6 ms     4 ms     3 ms  192.168.43.1
  2     *        *        *     Request timed out.
  3    39 ms    62 ms    33 ms  56.14.97.69
  4    42 ms    35 ms    30 ms  172.17.178.169
  5    63 ms    32 ms    37 ms  172.17.178.195
  6    71 ms    36 ms    34 ms  192.168.66.236
  7     *        *        *     Request timed out.
  8     *        *        *     Request timed out.
  9     *        *        *     Request timed out.
 10     *        *        *     Request timed out.
 11    57 ms    67 ms    64 ms  reliancejio.del01-96cbe-1b.ntwk.msn.net [104.44.196.187]
 12    62 ms    50 ms    48 ms  ae20-0.del01-96cbe-1a.ntwk.msn.net [104.44.42.87]
 13   106 ms    69 ms    60 ms  ae32-0.ear01.bom02.ntwk.msn.net [104.44.41.136]
 14   193 ms     *      122 ms  be-23-0.ibr03.bom02.ntwk.msn.net [104.44.34.35]
 15     *      300 ms    86 ms  be-5-0.ibr01.pnq21.ntwk.msn.net [104.44.30.142]
 16   203 ms    73 ms    70 ms  ae106-0.rwa04.pnq21.ntwk.msn.net [104.44.20.68]
 17     *        *        *     Request timed out.
 18     *        *        *     Request timed out.
 19     *        *        *     Request timed out.
 20     *        *        *     Request timed out.
 21     *        *        *     Request timed out.
 22     *        *        *     Request timed out.
 23     *        *        *     Request timed out.
 24     *        *        *     Request timed out.
 25     *        *        *     Request timed out.
 26     *        *        *     Request timed out.
 27     *        *        *     Request timed out.
 28     *        *        *     Request timed out.
 29     *        *        *     Request timed out.
 30     *        *        *     Request timed out.

Trace complete
```


and the result for the Test your connection


```
2507998 bytes downloaded from github.com at 9.39 Mbps
2507998 bytes downloaded from cloud.githubusercontent.com at 3.79 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 20.33 Mbps
2507998 bytes downloaded from github.githubassets.com at 9.46 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 5.04 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 3.55 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 14.33 Mbps
2507998 bytes downloaded from fra.github-debug.com at 5.46 Mbps
2507998 bytes downloaded from iad.github-debug.com at 3.82 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 4.38 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 6.58 Mbps
2507998 bytes downloaded from sea.github-debug.com at 3.49 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 8.66 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 4.73 Mbps

                
169.149.229.224

                
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

                
f10ab0a17781c913edb35f7a3999c821

                
F1BA:6C5B:06A9:2F40F:63B8FC09

                
                

                  
traceroute to 169.149.229.224 (169.149.229.224), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 169.149.229.224 (169.149.229.224) 56(84) bytes of data.

--- 169.149.229.224 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2027ms


                  
Start: 2023-01-06T21:19:18-0800
HOST: 311ba7fdeb3cc8e5b069f236b19dd063               Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
Host 224.229.149.169.in-addr.arpa. not found: 3(NXDOMAIN)
```


Looking at the results It's clear that I was experiencing 100% packet loss before my requests even reach GitHub's servers. Hence the issue with the ISP.

**Comment by izzybaer** [comment]
I am not experiencing any packet loss but my images are broken in my organizations profile readme

**Reply by MonitSharma** [reply]
Can you share the link to the readme?

**Reply by izzybaer** [reply]
I can't no, it's in a private organization. I can confirm with certainty that the image path is correct, referencing an image stored locally in the repo in an assets folder

**Reply by MonitSharma** [reply]
Go to the [debug site](https://github-debug.com/) and follow the guidelines there and paste the output here

**Reply by izzybaer** [reply]
izzybaer@izzybaer ~ % git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh

Cloning into '/tmp/debug-repo-ssh'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
izzybaer@izzybaer ~ % ping -c 10 github.com

PING github.com (140.82.114.3): 56 data bytes
64 bytes from 140.82.114.3: icmp_seq=0 ttl=54 time=26.517 ms
64 bytes from 140.82.114.3: icmp_seq=1 ttl=54 time=19.572 ms
64 bytes from 140.82.114.3: icmp_seq=2 ttl=54 time=26.225 ms
64 bytes from 140.82.114.3: icmp_seq=3 ttl=54 time=20.131 ms
64 bytes from 140.82.114.3: icmp_seq=4 ttl=54 time=25.784 ms
64 bytes from 140.82.114.3: icmp_seq=5 ttl=54 time=26.253 ms
64 bytes from 140.82.114.3: icmp_seq=6 ttl=54 time=19.289 ms
64 bytes from 140.82.114.3: icmp_seq=7 ttl=54 time=25.521 ms
64 bytes from 140.82.114.3: icmp_seq=8 ttl=54 time=26.303 ms
64 bytes from 140.82.114.3: icmp_seq=9 ttl=54 time=26.224 ms

--- github.com ping statistics ---
10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 19.289/24.182/26.517/2.976 ms

izzybaer@izzybaer ~ % traceroute github.com

traceroute to github.com (140.82.113.3), 64 hops max, 52 byte packets
 1  192.168.86.1 (192.168.86.1)  4.405 ms  2.723 ms  2.338 ms
 2  * * *
 3  * * *
 4  * * *
 5  * * *
 6  23-255-225-145.googlefiber.net (23.255.225.145)  14.221 ms  14.196 ms  13.984 ms
 7  * * *
 8  * * *
 9  eqix-dc5.github-1.com (206.126.237.27)  26.300 ms
    eqix-dc5.github-2.com (206.126.237.205)  20.224 ms
    eqix-dc5.github-1.com (206.126.237.27)  19.359 ms
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
31  * * *
32  * * *
33  * * *
34  * * *
35  * * *
36  * * *
37  * * *
38  * * *
39  * * *
40  * * *
41  * * *
42  * * *
43  * * *
44  * * *
45  * * *
46  * * *
47  * * *
48  * * *
49  * * *
50  * * *
51  * * *
52  * * *
53  * * *
54  * * *
55  * * *
56  * * *
57  * * *
58  * * *
59  * * *
60  * * *
61  * * *
62  * * *
63  * * *
64  * * *

**Comment by susantabiswas** [comment]
@MonitSharma  I am facing a similar issue and on Jio. Were you able to resolve this issue?

**Reply by MonitSharma** [reply]
It was seasonal, it didn't work for some time, but then suddenly one day it started working. I've now shifted from Jio, so I'm unsure how to correct it.  Make sure to check firewall, antivirus, low memory etc

**Comment by ahorisaac** [comment]
I'd have the same problem. Images in markdown files are not displaying as well as my profile image, and github users profile image(s).

I vividly followed the instructions provided by [github-debug](https://github-debug.com) and here is the output from my terminal.

I'm using a Windows PC.

`ping -n 10 github.com`

Pinging github.com [140.82.121.4] with 32 bytes of data:
Reply from 140.82.121.4: bytes=32 time=188ms TTL=47
Request timed out.
Request timed out.
Reply from 140.82.121.4: bytes=32 time=416ms TTL=47
Reply from 140.82.121.4: bytes=32 time=194ms TTL=47
Reply from 140.82.121.4: bytes=32 time=187ms TTL=47
Reply from 140.82.121.4: bytes=32 time=217ms TTL=47
Reply from 140.82.121.4: bytes=32 time=242ms TTL=47
Reply from 140.82.121.4: bytes=32 time=245ms TTL=47
Reply from 140.82.121.4: bytes=32 time=184ms TTL=47

Ping statistics for 140.82.121.4:
    Packets: Sent = 10, Received = 8, Lost = 2 (20% loss),
Approximate round trip times in milli-seconds:
    Minimum = 184ms, Maximum = 416ms, Average = 234ms

`tracert github.com`

Tracing route to github.com [140.82.121.4]
over a maximum of 30 hops:

  1   271 ms     2 ms     2 ms  192.168.1.1
  2   271 ms    46 ms    35 ms  10.208.142.66
  3   242 ms    36 ms    36 ms  10.208.142.65
  4     *      268 ms    45 ms  192.168.234.44
  5    46 ms    46 ms    48 ms  172.24.227.90
  6     *       68 ms    42 ms  192.168.50.2
  7     *        *        *     Request timed out.
  8     *        *        *     Request timed out.
  9   415 ms   166 ms   184 ms  be3184.ccr42.par01.atlas.cogentco.com [154.54.38.157]
 10   283 ms   196 ms   197 ms  prs-b6-link.ip.twelve99.net [213.248.86.169]
 11   185 ms   178 ms   177 ms  prs-bb1-link.ip.twelve99.net [62.115.125.118]
 12   504 ms   411 ms     *     ffm-bb1-link.ip.twelve99.net [62.115.123.12]
 13     *      210 ms   189 ms  ffm-b11-link.ip.twelve99.net [62.115.124.117]
 14   218 ms   188 ms   266 ms  github-ic-350972.ip.twelve99-cust.net [62.115.182.171]
 15     *        *        *     Request timed out.
 16     *        *        *     Request timed out.
 17   274 ms   205 ms   183 ms  lb-140-82-121-4-fra.github.com [140.82.121.4]

Trace complete.
